src/inmanta/plugins.py

```diff
@@ -172,6 +177,28 @@ def get_functions(cls) -> Dict[str, "Type[Plugin]"]:
         """
         return dict(cls.__functions)
 
+    @classmethod
+    def get_originals(cls) -> Dict[str, str]:
+        """
+        Get the original name of all the functions with the associated function name
+        """
+        return dict(cls.__function_name_mapping)
```

Comment:
I don't think this is used anymore.
```suggestion
    @classmethod
    def get_originals(cls) -> Dict[str, str]:
        """
        Get the original name of all the functions with the associated function name
        """
        return dict(cls.__function_name_mapping)
```